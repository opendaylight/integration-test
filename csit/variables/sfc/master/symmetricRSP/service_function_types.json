{
  "service-function-types": {
    "service-function-type": [
      {
        "type": "dpi-bidirectional",
        "bidirectional": "true",
        "nsh-aware": "true"
      },
      {
        "type": "dpi-non-bidirectional",
        "bidirectional": "false",
        "nsh-aware": "true"
      }
    ]
  }
}
