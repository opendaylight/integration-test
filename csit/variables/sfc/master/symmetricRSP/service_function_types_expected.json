{
  "service-function-types": {
    "service-function-type": [
      {
        "type": "dpi-bidirectional",
        "bidirectional": "true",
        "nsh-aware": "true",
        "sft-service-function-name": [
          {
            "name": "dpi-2"
          }
        ]
      },
      {
        "type": "dpi-non-bidirectional",
        "bidirectional": "false",
        "nsh-aware": "true",
        "sft-service-function-name": [
          {
            "name": "dpi-2"
          }
        ]
      },
      {
        "type": "firewall",
        "sft-service-function-name": [
          {
            "name": "firewall-1"
          }
        ]
      }
    ]
  }
}
